filament/backend/src/opengl/OpenGLDriver.h

```diff
@@ -256,6 +256,11 @@ class OpenGLDriver final : public DriverBase {
         return mHandleAllocator.handle_cast<Dp, B>(handle);
     }
 
+    template<typename B>
+    bool is_valid(Handle<B>& handle) {
```

Comment:
Is this getting called anywhere?

Reply:
so it's not, but it probably will in the texture view PR.